src/frontend/components/StudentLiveAdvertising/StudentLiveRegistration/RegistrationForm/index.tsx

```diff
@@ -108,9 +111,12 @@ export const RegistrationForm = ({
   const isSubmitOngoing = useRef(false);
 
   const onSubmit = async (email: string) => {
+    let anonymousId: Maybe<string>;
+    if (!checkLtiToken(getDecodedJwt())) {
```

Comment:
Is there any reason to omit non lti users ?

Reply:
to not assign them an anonymous_id ? If it's the question,  it's because they are not anonymous.